src/server/index.jsx

```diff
@@ -50,11 +50,26 @@ class LoggerStream {
   }
 }
 
-const constructDataFilePath = ({ pageType, service, id, variant = '' }) => {
-  const pageTypes = ['frontpage', 'mostRead'];
-  const dataPath = pageTypes.includes(pageType)
-    ? `${variant || 'index'}.json`
-    : `${id}${variant}.json`;
+const constructDataFilePath = ({
+  pageType,
+  service,
+  id,
```

Comment:
Can we rename `id` to `assetUri`? And delete the last param?

Reply:
@karinathomasbbc I thought it'd be useful to have the parameter be the same as the field in the [route regex](https://github.com/bbc/simorgh/pull/5718/files#diff-0e75337cdc4f57576789fbe816ebd2d7R51), since the `assetUri` is a path but `id` is just an id. This makes the usage make more sense, when reading the code.

Reply:
OK. Looks like articles don't have an assetUri (or assetUri is a CPS / TC2 thing only), so it would be more hassle to remove it.